avischiffmann/Coronavirus-Dashboard
# Sync USA data AND add override ability for any region.

## Summary

This is a cool one! It will scrape for more US data. It also compares both scrapers and picks the ones with the higher values because they are likely more updated.

Sadly CNN only has confirmed cases so the other values won't be updated but it is something. We can always add in more sources.

Also I added some JSON files in `overrides/`. If something is out of date you can update these files and since it passes through the thing that grabs the highest-value source then the override will show! Once the actual source is higher then it'll use the source instead. This should make manual updates much less tedious I hope. 😅 

**There is one TODO left for this pull request!** If you think its worth it then I am hoping you could go in and update the USA or Canada or whatever override file to the one that is in production right now.

I think that is easy? It should just be grabbing the JSON files you updated on master and putting it in the overrides folder. Since you did them I just wanted you to double check that logic because maybe I am wrong or something but I think thats right.

## Easy way to test Overrides
* Start the server
* In `overrides/` Edit any country's confirmed value to something ridic like 99,999
* Don't restart the server, just wait a minute
* Refresh the page and that value should be on the page and in the table with the TOTAL row recalculated too.
